conversation_reconstruction_dataflow/conversation_reconstruction/construct_utils/conversation_constructor.py

```diff
@@ -36,22 +39,25 @@ def insert(rev, page, previous_comments, DEBUGGING_MODE = False):
     removed_actions = {}
     old_actions = sorted(page['actions'].keys())
     modification_actions = defaultdict(int)
-    rev_text = text_split.tokenize('\n' + rev['text'])
+    rev_text = text_split.tokenize(rev['text'])
     for op in rev['diff']:
         if DEBUGGING_MODE : 
```

Comment:
I'm not sure how much output this creates, but I expect a lot. Probably good to have a notion of log-every-n-statements, and use something like hash modulo log-every-n-statements to allow us to print out 1/100 comments instead of each one.  Probably good to abstract into a debug_log_every_n function. But you can do this in a followup PR if you like.

Reply:
Not sure if I understand this correctly, I'm currently output 1 of every 1000 output to the logging information on dataflow console for debugging inspection, would that be the thing you are suggesting?